ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -1731,8 +1731,8 @@ def verify_multus_network():
         ]
         if public_net_created:
             assert (
-                public_net_full_name in pod_networks
-            ), f"{public_net_full_name} not in {pod_networks}"
+                public_net_name in pod_networks
```

Comment:
Should we be a bit more explicit here? This no longer verifies that the namespace is correct. Since the new formatting is as such: 

```
k8s.v1.cni.cncf.io/networks: '[{"name":"public-net","namespace":"openshift-storage"},{"name":"private-net","namespace":"openshift-storage"}]'
```

We could do something like this:

```
networks = [{"name":"public-net","namespace":"openshift-storage"},{"name":"private-net","namespace":"openshift-storage"}]  # would need to convert pod_networks to an actual list like this

assert(any((network["name"] == public_net_name and network["namespace"] == public_net_namespace) for network in networks))

```
You'll need to convert the networks string into an actual list to parse it like that but this way we don't lose the ability to verify that the namespace is correct.

Reply:
done. 
@clacroix12 Can you check my fix?